src/net.h

```diff
@@ -1071,6 +1071,7 @@ class CConnman
         int64_t m_peer_connect_timeout = DEFAULT_PEER_CONNECT_TIMEOUT;
         std::vector<std::string> vSeedNodes;
         std::vector<NetWhitelistPermissions> vWhitelistedRange;
+        std::vector<NetWhitelistPermissions> vWhitelistedRangeOutgoing;
```

Comment:
Might be nice for future devs looking at this code to rename the original `vWhitelistedRange` to `vWhitelistedRangeIncoming`, unless there's a reason like is it ever not just for incoming connections?

Reply:
Done